<!DOCTYPE html>

<html>
<head>
<meta charset="utf-8">
<title>Research</title>
</head>
<link rel="stylesheet" href="style.css">
<body  id="gradientbkrd2" class="fade-in" style="color:gray;">
<div id="bottommargin">



<h1>Research</h1>




	
	

<div id="main">

    <h2> Codes of promoter level transcription regulation in development </h2>
	
	 
	 <!--img class="mi-img" src="images\website material\Promoter architecture during the course of the cell cycle\tri_fucci.png" --> 
	 
	 <video class="mi-img" width="640" height="640" controls>
	<source src="images\website material\Promoter architecture during the course of the cell cycle\fuccivid-new.mp4" type=video/mp4 >
	
		</video>
		
		
	 <p> The core promoter is a DNA sequence, which is required for recruitment of general transcription factors and provide the platform for Polymerase II transcription. Promoters come in many shapes and sizes and contribute to transcription regulation and serve as an integration point for diverse signals conveyed by cis regulatory elements such as enhancers. The diversity of core promoter architectures with distinct transcription initiation profiles in vertebrate genomes points at an unexplained regulatory level. Despite of decades of study, we are still unable to clearly define the DNA sequences, which determine where and how Polymerase II initiates transcription. We combine genomics approaches together with functional studies in an embryo model to understand promoter level gene regulation during developmental of the vertebrate embryo.<br><br>
 </p>
	
	<h2> Transcription, chromatin dynamics and nuclear organisation in early development </h2>
	

		
	<p> Animal bodies including that of humans develop from a single cell, the fertilized egg. The development of embryos from fertilized eggs starts through a dramatic changeover from the uniquely differentiated large cell to a fast dividing mass of embryonic cells or stem cells. These stem cells undergo a sequence of changes, which leads them to form cellular lineages and eventually a variety of tissues. This process is governed by the coordinated activation of thousands of genes at the right time and in the right cells through gene regulation. When and where genes get switched on is a key determinant of normal development and is encoded in the DNA (genome) sequence. There is an additional layer of gene control, which operates by selective and dynamic packaging of DNA into chromatin. Chromatin states and conformation vary in development and between the cells of the embryo and defines the so called epigenetic regulation of gene activation (transcription). We study how the earliest genes get switched on in the embryo, how are they organised in the nucleus of embryonic cells and how epigenetic mechanisms influence the sequential regulation of genes during embryo development. We study how the control of early development which is initiated by signals deposited by the mother into the fertilised egg in the form of RNAs, proteins and chromatin packaging give way to control of development by the embryoâ€™s own genes. We use a small animal model zebrafish, which is popular for genetic analysis of development due to the similarities in the way fish and human embryos regulate their genes and because the small transparent fish embryos can be easily cultured and monitored in laboratory conditions. We use biochemical assays and next generation sequencing methods to understand the mechanisms of gene regulation during early embryo development. We thus aim to contribute to the understanding the regulation of genes in normal development which will also inform on the reasons for abnormal gene regulation, which causes a variety of disease both in development and in adult life. <br><br>
	
		<video class="mi-img" width="640" height="480" controls>
	<source src="images\website material\Imaging transcription\TwitterMovie_5fps.mp4" type=video/mp4 style="width:100%">
	
		</video>
		
 <i> Transcription imaging in live embryos </i><br><br>
 Core promoters and their interaction with cis regulatory modules regulate spatio-temporal dynamics of gene expression and explain not only lineage and tissue specificity but also cell to cell variation of gene expression. To get insight into regulatory principles of transcription dynamics on the single gene and single cell level, we develop transcription imaging tools. Our recent work with a new imaging approach, which we call MOVIE, allows native nascent transcription detection in the transparent zebrafish embryo. Using MOVIE we momitored the transcription dynamics of the first gene expression in the zebrafish embryo and study the nuclear organisation of the earliest gene expression in unique nuclear compartments. 
 <br><br>
 </p>
	
	<h2> Biomedical application of the zebrafish model </h2>
	<img class="midimg" src="images\website material\placeholders\image009.gif">
	<p> Cancer is a highly heterogeneous set of diseases, even between those classified as the same cancer type. This is particularly true in their responses to therapeutics. There is therefore an urgent need to identify patients that will respond best to different therapeutic options. Work in our lab has identified a novel molecular profile, associated with the way in which transcription starts, which is predictive of tumour response to radiotherapy and other stressors. We currently have multiple projects running with aims to characterise this molecular profile, with a particular focus on whether it can be modified to sensitize tumours to radiotherapy. This work involves tumour organoids, zebrafish xenograft modelling, Cap Analysis of gene expression (CAGE), chromatin immunoprecipitation (ChIP), Assay for Transposase-Accessible Chromatin (ATAC) and 4D imaging approaches.<br><br>
 </p>
	
	<h2> Disease-associated cis-regulatory elements in developmental gene regulation  </h2>
	<img class="midimg" src="images\website material\placeholders\firsttry.jpg">
	<p> The aberration of transcription regulation of genes can lead to congenital and multifactorial diseases. Large-scale genomics programmes such as ENCODE and FANTOM resulted in prediction of previously unanticipated density of functional elements of the human genome. These predictions raise the need for validation models of predicted functional elements. We are assessing the degree to which zebrafish with the transparent, externally developing embryo can be used as a surrogate for validating cis-regulatory functions predicted by genome wide assays in mammals and in fish. We develop methods for functional analysis of predicted cis regulatory elements associated with disease.<br><br>
 </p>
	
	
</div>



	 <div id="sidenav">
	 
	 
  <a href="index.html">Mueller lab</a>
  <a href="lab_members.html">Lab Members</a>
  <a href="research.html">Research</a>
  <a href="publications.html">Publications</a>
  <a href="thelab.html">The Lab</a>
  <a class="mntxt" href="resources.html">Resources</a>
  <a href="contact.html">Contact/Join us</a>

   
<script src="myscripts.js"></script>


   
  

   <a class="twitter-timeline" data-width="250" data-height="400" href="https://twitter.com/LabMueller?ref_src=twsrc%5Etfw">Tweets by LabMueller</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script> 
	<br>
	<a href="https://twitter.com/LabMueller?ref_src=twsrc%5Etfw" class="twitter-follow-button" data-size="large" data-show-count="false">Follow @LabMueller</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>


 <div>
	 <img src="images\website material\ICGS.png" width=75% > 
	 <br>
	 <img src="images\website material\wellcome.jpg" width=75% > 
	 </div>
   </div>
     <footer class="footer">
  <p>Mueller lab, 2020</p>
</footer> 
</div>
  

</body> 
</html>