<!DOCTYPE html>

<html>
<head>
<meta charset="utf-8">
<title>Research</title>
</head>
<link rel="stylesheet" href="style.css">
<body  id="gradientbkrd2" class="fade-in" style="color:gray;">
<div id="bottommargin">



<h1>Research</h1>




	
	

<div id="main">

    <h2> Codes of promoter level transcription regulation in development </h2>
	
	 
	 <!--img class="mi-img" src="images\website material\Promoter architecture during the course of the cell cycle\tri_fucci.png" --> 
	 
	 <video class="mi-img" width="640" height="480" controls>
	<source src="images\website material\Promoter architecture during the course of the cell cycle\fuccivid.mp4" type=video/mp4 >
	
		</video>
		
		
	 <p> The core promoter is a DNA sequence, which is required for recruitment of general transcription factors and provide the platform for Polymerase II transcription. Promoters come in many shapes and sizes and contribute to transcription regulation and serve as an integration point for diverse signals conveyed by cis regulatory elements such as enhancers. The diversity of core promoter architectures with distinct transcription initiation profiles in vertebrate genomes points at an unexplained regulatory level. Despite of decades of study, we are still unable to clearly define the DNA sequences, which determine where and how Polymerase II initiates transcription. We combine genomics approaches together with functional studies in an embryo model to understand promoter level gene regulation during developmental of the vertebrate embryo.<br><br>
 </p>
	
	<h2> Transcription, chromatin dynamics and nuclear organisation in early development </h2>
	
	<video class="mi-img" width="640" height="480" controls>
	<source src="images\website material\Imaging transcription\TwitterMovie_5fps.mp4" type=video/mp4 style="width:100%">
	
		</video>
		
	<p> Early development of zebrafish encompassing the maternal to zygotic transition and including zygotic genome activation provides an ideal experimental platform for elucidating the epigenomic features of gene regulation. It allows dissecting the temporal sequence and dynamics of establishing transcriptionally active chromatin state and helps in identifying the determinants of transcription activation of Polymerase II transcribed genes. The relatively large number of pluripotent cells generated by the fast cell divisions prior to zygotic transcription provides sufficient biomass for next generation sequencing technology approaches to establish the temporal dynamics of transcription regulatory events and suggest causative relationship between them. We are annotating epigenetic features, transcriptional start site regions and distal cis-regulatory modules such as enhancers of the developing embryo and its various lineages by CAGE-seq, ChIP-seq ATAC-seq and RNA-seq during development. Analyses of these epigenomic datasets strongly support a model which suggest that epigenomic premarking mechanisms contribute to developmental gene regulation programme well ahead of commencement of the activation of gene activation and that gametic epigenetic information may contribute to embryonic developmental expression programmes.<br><br>
 <i> Transcription imaging in live embryos </i><br><br>
 Core promoters and their interaction with cis regulatory modules regulate spatio-temporal dynamics of gene expression and explain not only lineage and tissue specificity but also cell to cell variation of gene expression. To get insight into regulatory principles of transcription dynamics on the single gene and single cell level, we develop transcription imaging tools. Our recent work with a new imaging approach, which we call MOVIE, allows native nascent transcription detection in the transparent zebrafish embryo. Using MOVIE we momitored the transcription dynamics of the first gene expression in the zebrafish embryo and study the nuclear organisation of the earliest gene expression in unique nuclear compartments. 
 <br><br>
 </p>
	
	<h2> Biomedical application of the zebrafish model </h2>
	<img class="midimg" src="images\website material\placeholders\image009.gif">
	<p> The Mueller group offers their functional genomic experience with the fish model to collaborate with clinical and non-clinical investigators in their search for the in vivo function of disease causing genes. Based on their experience with embryo phenotyping screening the Mueller group also collaborate with investigators in exploiting zebrafish in developing tools for physiological phenotype detection for screening for drug effects (therapeutic or toxicity) and they seek adapting the embryo phenotyping technologies in non-embryo models such as cancer organoids.<br><br>
 </p>
	
	<h2> Disease-associated cis-regulatory elements in developmental gene regulation  </h2>
	<img class="midimg" src="images\website material\placeholders\firsttry.jpg">
	<p> The aberration of transcription regulation of genes can lead to congenital and multifactorial diseases. Large-scale genomics programmes such as ENCODE and FANTOM resulted in prediction of previously unanticipated density of functional elements of the human genome. These predictions raise the need for validation models of predicted functional elements. We are assessing the degree to which zebrafish with the transparent, externally developing embryo can be used as a surrogate for validating cis-regulatory functions predicted by genome wide assays in mammals and in fish. We develop methods for functional analysis of predicted cis regulatory elements associated with disease.<br><br>
 </p>
	
	
</div>



	 <div id="sidenav">
	 
	 
  <a href="index.html">Mueller lab</a>
  <a href="lab_members.html">Lab Members</a>
  <a href="research.html">Research</a>
  <a href="publications.html">Publications</a>
  <a class="mntxt" href="resources.html">Resources</a>
  <a href="contact.html">Contact/Join us</a>


   
<script src="myscripts.js"></script>
   
   <a class="twitter-timeline" data-width="250" data-height="400" href="https://twitter.com/LabMueller?ref_src=twsrc%5Etfw">Tweets by LabMueller</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script> 
	<br>
	<a href="https://twitter.com/LabMueller?ref_src=twsrc%5Etfw" class="twitter-follow-button" data-size="large" data-show-count="false">Follow @LabMueller</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
   
</div>
     <footer class="footer">
  <p>Mueller lab, 2020</p>
</footer> 
</div>
  

</body> 
</html>